The latest version v7.0.0-rc.1 does not build

## 🐛 Bug Report

The latest version v7.0.0-rc.1 does not build

## To Reproduce

Steps to reproduce the behavior:
Include the module by running the following command
`npm install @elastic/elasticsearch --save`

## Expected behavior

Should build without any errors

However i get the following error 
`> tsc

node_modules/@elastic/elasticsearch/lib/Connection.d.ts:77:3 - error TS1165: A computed property name in an ambient context must refer to an expression whose type is a literal type or a 'unique symbol' type.

77   [inspect.custom](object: any, options: InspectOptions): string;
     ~~~~~~~~~~~~~~~~
`

## Your Environment

- *node version*: 8
- *@elastic/elasticsearch version*: >=7.0.0
- *os*: Mac
- *typescript*: 3.4.1
- *tslint*: 5.14.0

## Other observations

Including @types/elasticsearch would solve the issue however the types support is only for version 5.0.23

I just tried updating node (v10.15.3) and ran the same commands you listed, but am seeing the same error:

```
node_modules/@elastic/elasticsearch/lib/Connection.d.ts:77:3 - error TS1165: A computed property name in an ambient context must refer to an expression whose type is a literal type or a 'unique symbol' type.

77   [inspect.custom](object: any, options: InspectOptions): string;
```

Note that it compiled if you run `tsc index.ts` immediately after `touch index.ts`, but throws the error after I add the actual **index.ts** code that includes the `import`.

--- a/lib/Connection.d.ts
+++ b/lib/Connection.d.ts
@@ -76,6 +76,7 @@ export default class Connection {
   setRole(role: string, enabled: boolean): Connection;
   status: string;
   buildRequestObject(params: any): http.ClientRequestArgs;
+  // @ts-ignore
   [inspect.custom](object: any, options: InspectOptions): string;
 }
 

